src/html/mod.rs

```diff
@@ -263,6 +266,28 @@ where
     }
 }
 
+/// Wrapped Element reference for later use in Component lifecycle methods.
```

Comment:
I think a proper example section here would be useful. Eg. explain/show that this can be used to set the `ref` framework-attribute in a component/tag.

Alternatively, adding link to the relevant example might work as well.